packages/language-service/src/completions.ts

```diff
@@ -531,6 +537,62 @@ export class CompletionBuilder<N extends TmplAstNode|AST> {
     return directive?.tsSymbol;
   }
 
+  private isAnimationCompletions(): this is ElementAnimationCompletionBuilder {
+    return (this.node instanceof TmplAstBoundAttribute &&
+            this.node.type === BindingType.Animation) ||
+        (this.node instanceof TmplAstBoundEvent && this.node.type === ParsedEventType.Animation);
+  }
+
+  private getAnimationCompletions(this: ElementAnimationCompletionBuilder):
+      ts.WithMetadata<ts.CompletionInfo>|undefined {
+    if (this.node instanceof TmplAstBoundAttribute) {
+      const animations = this.compiler.getTemplateTypeChecker()
+                             .getDirectiveMetadata(this.component)
+                             ?.animationTriggerNames?.triggerNames;
+      const replacementSpan = makeReplacementSpanFromParseSourceSpan(this.node.keySpan);
+      if (animations) {
+        const entries = buildAnimationCompletionEntries(
+            [...animations, '.disabled'], replacementSpan, DisplayInfoKind.ATTRIBUTE);
+        return {
+          entries,
+          isGlobalCompletion: false,
+          isMemberCompletion: false,
+          isNewIdentifierLocation: true,
+        };
+      }
+    } else {
+      const animationNameSpan = buildAnimationNameSpan(this.node);
+      const phaseSpan = buildAnimationPhaseSpan(this.node);
+      if (isWithin(this.position, animationNameSpan)) {
+        const animations = this.compiler.getTemplateTypeChecker()
+                               .getDirectiveMetadata(this.component)
+                               ?.animationTriggerNames?.triggerNames;
+        const replacementSpan = makeReplacementSpanFromParseSourceSpan(animationNameSpan);
+        if (animations) {
+          const entries =
+              buildAnimationCompletionEntries(animations, replacementSpan, DisplayInfoKind.EVENT);
+          return {
+            entries,
+            isGlobalCompletion: false,
+            isMemberCompletion: false,
+            isNewIdentifierLocation: true,
+          };
+        }
+      }
+      if (phaseSpan !== null && isWithin(this.position, phaseSpan)) {
+        const replacementSpan = makeReplacementSpanFromParseSourceSpan(phaseSpan);
+        const entries = buildAnimationCompletionEntries(
+            ['start', 'done'], replacementSpan, DisplayInfoKind.EVENT);
```

Comment:
Let's extract this array to a constant that can be self-documenting with its name.
```
const ANIMATION_PHASES = ['start', 'done'];
```

```suggestion
            ANIMATION_PHASES, replacementSpan, DisplayInfoKind.EVENT);
```